src/sdk/bucketspace_nb.js

```diff
@@ -189,15 +190,24 @@ class BucketSpaceNB {
             _.isUndefined(account.nsfs_account_config.gid)) return false;
         try {
             dbg.log0('_has_access_to_nsfs_dir', namespace_bucket_config.write_resource, account.nsfs_account_config.uid, account.nsfs_account_config.gid);
+            // always allow root access
+            if (account.nsfs_account_config.uid === 0 && account.nsfs_account_config.gid === 0) return true;
 
-            await nb_native().fs.stat({
+            const stat = await nb_native().fs.stat({
                     uid: account.nsfs_account_config.uid,
                     gid: account.nsfs_account_config.gid,
                     backend: namespace_bucket_config.write_resource.resource.fs_backend
-                }, namespace_bucket_config.write_resource.resource.fs_root_path);
+                }, path.join(namespace_bucket_config.write_resource.resource.fs_root_path, namespace_bucket_config.write_resource.path || ''));
 
-            return true;
+            if (!stat) return false;
+
+            const octal_others_mode = parseInt(stat.mode.toString(8), 10) % 10;
```

Comment:
isn't it simpler to use modulo 8?
```suggestion
            const octal_others_mode = stat.mode % 8;
```

Reply:
yes, fixed it thanks.